server/src/routes/email_routes.py

```diff
@@ -0,0 +1,24 @@
+from fastapi import FastAPI, APIRouter
+import smtplib
+from email.message import EmailMessage
+from email.mime.text import MIMEText
+from pydantic import BaseModel
+
+router = APIRouter()
+
+class EmailRequest(BaseModel):
+    body: str
+    recipient: str
+
+@router.post("/send_email")
+def send_email(request: EmailRequest):
+
+    msg = MIMEText(request.body)
```

Comment:
![image](https://github.com/SIS-Team-24/2023-SIS-Team-24/assets/44883385/00a23f30-07ed-446f-92d9-915fc480f287)

Email has the main things noiceeee 👌. Just some minor styling things:
1. New line between "Here is your text summary:" and the actual summary. (probs chuck a `\n` in between them)
2. At the bottom of the text, a sign off or smth 🤷 to make the email more email-y (another `\n` and then sign off) 
3. Perhaps, idek how easy it'll be but, try and add the logo into the email (Not too fussed bc it might be hard 🤷 )